pkg/authz/wookie/mysql.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2023 Forerunner Labs, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package authz
+
+import (
+	"context"
+
+	"github.com/pkg/errors"
+	"github.com/warrant-dev/warrant/pkg/database"
+)
+
+type MySQLRepository struct {
+	database.SQLRepository
+}
+
+func NewMySQLRepository(db *database.MySQL) *MySQLRepository {
+	return &MySQLRepository{
+		database.NewSQLRepository(&db.SQL),
+	}
+}
+
+func (repo MySQLRepository) Create(ctx context.Context, version int64) (int64, error) {
+	result, err := repo.DB.ExecContext(
+		ctx,
+		`
+			INSERT INTO wookie (
+				ver
```

Comment:
Can you explicitly set `createdAt` to `CURRENT_TIMESTAMP(6)`

Reply:
Should I also update all other `Create` queries to do the same?

Reply:
No, let's just do it on this one for now.